m4_sessions/1808-inning/prejudices.md

```diff
@@ -13,9 +16,11 @@ Turing's mission is to unlock human potential by training a diverse, inclusive s
 (What will students learn/leave with?): 
 
 ## Session Outline 
-
-(Please include who is facilitating and how much time you are going to spend in each section):
-
+* Intros (10-15 mins) led by entire group. Includes rules for how conversations will elvolve to make a safe space for discussion
+* Implicit Bias Tests (10 mins) Students should bring a computer and we will slack them the link for implicit bias tests with instructions to take a test on a topic that they either know very little about or something they feel very 'woke' to.  We haven't decided which way to lean.
+* Small Group Discussion (15 mins) Break into 3 small groups, each facilitated by a group member to discuss results, reactions to results, etc
```

Comment:
This is awesome.  Please add some discussion questions you plan on using for this part.  It would be nice for each of the facilitators to have a "go to" set of questions.